docs/configs.md

```diff
@@ -142,6 +142,7 @@ Name | Description | Default Value
 <a name="sql.udfCompiler.enabled"></a>spark.rapids.sql.udfCompiler.enabled|When set to true, Scala UDFs will be considered for compilation as Catalyst expressions|false
 <a name="sql.variableFloatAgg.enabled"></a>spark.rapids.sql.variableFloatAgg.enabled|Spark assumes that all operations produce the exact same result each time. This is not true for some floating point aggregations, which can produce slightly different results on the GPU as the aggregation is done in parallel.  This can enable those operations if you know the query is only computing it once.|true
 <a name="sql.window.range.byte.enabled"></a>spark.rapids.sql.window.range.byte.enabled|When the order-by column of a range based window is byte type and the range boundary calculated for a value has overflow, CPU and GPU will get the different results. When set to false disables the range window acceleration for the byte type order-by column|false
+<a name="sql.window.range.decimal.enabled"></a>spark.rapids.sql.window.range.decimal.enabled|Support for DECIMAL in range based window functions is tentative. When set to false disables the range window acceleration for the DECIMAL type order-by column|true
```

Comment:
How is this tentative? Kind of confused if CUDF does not support it then shouldn't this be a draft PR. If it does, do you just not feel confident in it?

Reply:
I meant to replace that with "experimental", because it's new. Should I just remove that line?

Reply:
No experimental I understand and is fine. Great work.

Reply:
I've removed the line. Regenerating documentation now.